bots/tests-scan

```diff
@@ -88,6 +88,11 @@ REPO_BRANCH_CONTEXT = {
         ],
         'rhel-8.1': ['cockpit/rhel-8-1/chrome', 'cockpit/rhel-8-1/firefox', 'cockpit/rhel-8-1/edge'
         ],
+    },
+    'mvollmer/subscription-manager': {
+        'master': [
+            'cockpit/rhel-8-0'
+        ]
```

Comment:
Normally I'd start with the `_manual` branch, as otherwise all of your PRs would instantly get red. But as this is a demo branch only, I'm ok with immediately running on master.

Reply:
I have changed it to "_manual".